roverrobotics_driver/src/roverrobotics_ros_driver.cpp

```diff
@@ -115,23 +141,23 @@ ROSWrapper::ROSWrapper(ros::NodeHandle *nh) {
     ROS_INFO("no 'status_topic' set; using the default value: '/robot_status'");
     robot_status_topic_ = "/robot_status";
   }
-  if (!ros::param::get("status_frequency", robot_status_frequency)) {
+  if (!ros::param::get("status_frequency", robot_status_frequency_)) {
     ROS_INFO("no 'status_frequency' set; using the default value: '5.00'");
-    robot_status_frequency = 5.00;
+    robot_status_frequency_ = 5.00;
```

Comment:
same notes, variable defaults should generally not be defined in any .c or .cpp files.